styles/styles.css

```diff
@@ -578,23 +578,23 @@ main > .section.small-screen-cta-container > .grid-container {
 .grid-container--with-scroll {
   --card-drop-shadow-size-x: var(--spectrum-drop-shadow-blur-100);
   --card-drop-shadow-size-y: calc(var(--spectrum-drop-shadow-y-100) + var(--spectrum-drop-shadow-blur-100));
-
-  overflow-x: scroll;
+ 
   padding-block: var(--card-drop-shadow-size-y);
   padding-inline-start: var(--card-drop-shadow-size-x);
   margin-inline-start: calc(var(--card-drop-shadow-size-x) * -1);
-
-  @media (min-width: 36rem) {
-    overflow-x: unset;
+ 
+  @media not all and (min-width: 36rem) {
+    overflow-x: scroll;
+    margin-inline-end: calc(var(--page-gutters) * -1);
+    padding-inline-end: var(--page-gutters);
+    max-width: calc(100% + var(--page-gutters));
   }
 
   @supports (display: grid) {
-    grid-auto-flow: column;
-    grid-auto-columns: var(--horizontal-scroll-block-width);
-
-    @media (min-width: 36rem) {
-      grid-auto-flow: unset;
-      grid-auto-columns: unset;
+    @media not all and (min-width: 36rem) {
+      grid-auto-flow: column;
+      grid-auto-columns: var(--horizontal-scroll-block-width);
+      grid-template-columns: none;
```

Comment:
This line fixes the extra space at the end of the horizontal scroll, because of the inherited value. And the explicit columns clashing with the implicit auto columns. Weird one.

I reorganized these media queries to use `not`, so we only need one range and don't need to unset and reset values.